### ISSUE TITLE
Godot 4.0 RC6 VehicleWheel3D nodes do not collide with FBX generated ConcavePolygonShape3D Godot

### ISSUE THREAD

**Issue by hudmarc** [godotengine/godot#74083]
### Godot version

4.0 RC 6

### System information

macOS Intel

### Issue description

VehicleWheel3D nodes do not  seem to collide with ConcavePolygonShape3D colliders generated from a concave, high poly FBX terrain

Test cases:

- `ConcavePolygonShape3D` + imported low poly `gLTF 2.0 (.glb)` convex terrain: 

> VehicleWheel3D collides as expected

- `ConcavePolygonShape3D` + imported low poly `FBX` convex terrain : 

> VehicleWheel3D collides as expected

- `ConcavePolygonShape3D` + imported high poly `gLTF 2.0 (.glb)` convex terrain : 

> VehicleWheel3D collides as expected

- `ConcavePolygonShape3D` + imported .high poly `FBX` convex terrain : 

> VehicleWheel3D collides as expected

- `ConcavePolygonShape3D` + imported high poly `gLTF 2.0 (.glb)` convex terrain with messed up normals : 

> VehicleWheel3D collides as expected (falls through holes in terrain as expected)

- `ConcavePolygonShape3D` + imported .high poly `FBX` convex terrain  with messed up normals: 

> VehicleWheel3D collides as expected (falls through holes in terrain as expected)

- `ConcavePolygonShape3D` + imported high poly `gLTF 2.0 (.glb)` **concave** terrain: 

> VehicleWheel3D collides as expected

- `ConcavePolygonShape3D` + imported high poly `FBX` **concave** terrain: 

> **_VehicleWheel3D does not collide as expected, wheels do not interact with terrain at all_**

### Steps to reproduce

Create a simple vehicle, place it on an imported convex FBX terrain, with the ConcavePolygonShape3D mesh generated using `Mesh > Create Trimesh Static Body`

### Minimal reproduction project

[GodotCarTest.zip](https://github.com/godotengine/godot/files/10844371/GodotCarTest.zip)

**Comment by Calinou** [comment]
You've left the suspension travel to the default value in 4.0, which is way too high: 
![image](https://user-images.githubusercontent.com/180032/226699624-72ab088c-6025-4094-9d38-f88ed3e70543.png)

This was changed to 0.2 in 4.0.1, but you can try adjusting the value between 0.1 and 0.3 in general.

**Comment by hudmarc** [comment]
Yup, but that isn't the problem as far as I can tell. Regardless of the setting for Travel wheels simply do not function when colliding with a concave mesh collider generated from an imported FBX, so I've had to re-export all terrains as gLTF files in order to get functioning collision.
